build_tools/benchmarks/benchmark_helper.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env python3
+# Copyright 2023 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+"""Miscellaneous tool to help work with benchmark suite and benchmark CI."""
+
+import pathlib
+import sys
+
+# Add build_tools python dir to the search path.
+sys.path.insert(0, str(pathlib.Path(__file__).parent.with_name("python")))
+
+import argparse
+import json
+from typing import List
+
+from e2e_test_framework import serialization
+from e2e_test_artifacts import model_artifacts, iree_artifacts
+from e2e_test_framework.definitions import iree_definitions
+
+
+def dump_flags_of_generation_config(
+    module_generation_config: iree_definitions.ModuleGenerationConfig,
+    root_path: pathlib.PurePath = pathlib.PurePath()):
+
+  imported_model = module_generation_config.imported_model
+  imported_model_path = iree_artifacts.get_imported_model_path(
+      imported_model=imported_model, root_path=root_path)
+  source_model_path = model_artifacts.get_model_path(model=imported_model.model,
+                                                     root_path=root_path)
+
+  compile_flags = module_generation_config.materialize_compile_flags() + [
+      str(imported_model_path)
+  ]
+  import_flags = imported_model.import_config.materialize_import_flags(
+      model=imported_model.model) + [str(source_model_path)]
```

Comment:
We probably should also have a full flag list in `iree_definitions.ImportedModel`, just like we have done with `ModuleGenerationConfig` and `E2EModelRunConfig`. Will do a follow-up change.